### PR TITLE
More code and script change to enable vFile CI

### PR DESCRIPTION
**Pull request by luomiao**
@ashahi1 
Please generate the key. Thanks!

Update:
Due to the slowness of the testbed, CI testing exposed a few problems in the code.
Besides adding scripts change, this PR also includes several fixes:
1. Change the waiting for file server path mounting from one time waiting to a timeout waiting. -> Fixes: #1941 #1816 
2. When some of the nodes in the swarm network failed/rejected for an assigned docker service task, the status of this task will stay in task lists if polling information according to service ID. Current code will return false as long as one task is failed/rejected. However the correct handling should return true as long as one task is running, since we only need one running container for the file server service.  Fixes: #1911

### CODE REVIEW

**Comment by ashahi1** [comment]
CI run has started with the new key.

File: .drone.yml [hunk 1]
```diff
@@ -49,6 +49,7 @@ build:
       - go get github.com/golang/lint/golint
       - go get -u gopkg.in/check.v1
       - make -s build
+      - make -s build-vfile
```
Comment (shuklanirdesh82) [inline .drone.yml]: it is better to include the target into the `build` target it self

Comment (luomiao) [reply .drone.yml]: If we include vFile also into build target, all the following development for vDVS will suffer waiting for building vfile too. The build of vfile and vdvs are totally separate from each other. So I think we should have them as different build target.

Comment (shuklanirdesh82) [reply .drone.yml]: building vFile binary wont take much time it is still OK to build binary as part of build having it said building and pushing plugin should be a separate target and I think it is there. Am I missing anything here?

**Comment by luomiao** [comment]
@shuklanirdesh82 
The updated PR has addressed Nirdesh's comment to include vfile as part of build target.
Since Nirdesh is PTO now, I will just dismiss this review since it's already cleared.

**Comment by luomiao** [comment]
@lipingxue 
since there are some changes to vfile code too, could you also please review those changes?
Thanks.

**Comment by lipingxue** [comment]
@luomiao OK. I will review it.

**Pull request merged**